controllers/podchaos/containerkill/types.go

```diff
@@ -88,6 +89,7 @@ func (r *endpoint) Apply(ctx context.Context, req ctrl.Request, obj v1alpha1.Inn
 		}
 
 		if haveContainer == false {
+			allHaveContainer = false
 			r.Log.Error(nil, fmt.Sprintf("the pod %s doesn't have container %s", pod.Name, podchaos.Spec.ContainerName))
```

Comment:
We should return this error here

Reply:
@cwen0
Then there will be a situation that is, if we have pod1, pod2, and there is a specified container in pod2, if we traverse pod1 first, and then return an error directly here, it is possible that pod2 is not processed.